internal/cmdsearch/cmdsearch.go

```diff
@@ -57,11 +57,6 @@ func NewSearchRunner(name string) *SearchRunner {
 	return r
 }
 
-const shortMessage = `Search and optionally replace fields across all resources. 
```

Comment:
Why remove this?

Reply:
They are auto-generated from mdtogo and added here. https://github.com/GoogleContainerTools/kpt/pull/1456/files#diff-d16bb5b078337f35bfbb11af3240a9477e2d7b5972739f56cf9b137beb90d7d9R109